airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -268,23 +269,15 @@ private JsonNode getColumnMetadata(final ResultSet resultSet) throws SQLExceptio
         .build());
   }
 
-  /**
-   * @param field Essential column information returned from
-   *        {@link AbstractJdbcSource#getColumnMetadata}.
-   */
-  private Datatype getFieldType(final JsonNode field) {
-    return sourceOperations.getFieldType(field);
-  }
-
   @Override
   public List<TableInfo<CommonField<Datatype>>> discoverInternal(final JdbcDatabase database)
       throws Exception {
     return discoverInternal(database, null);
   }
 
   @Override
-  public JsonSchemaType getType(final Datatype columnType) {
-    return sourceOperations.getJsonType(columnType);
+  public JsonSchemaType getAirbyteType(final Datatype columnType) {
```

Comment:
Would it make sense to also get rid of this method since it's similar to the removal of `getFieldType` above in that it's mainly calling `sourceOperations.get<Airbyte>Type`?

Reply:
This is defined in `AbstractDbSource` and is overloaded by a couple of different connectors. I think this is doable, but prefer not to increase the scope of this refactor